src/main/scala/gitbucket/core/controller/RepositoryViewerController.scala

```diff
@@ -848,15 +848,14 @@ trait RepositoryViewerControllerBase extends ControllerBase {
     redirect(s"${repository.owner}/${repository.name}/releases")
   })
 
-  get("/:owner/:repository/archive/:name")(referrersOnly { repository =>
-    val name = params("name")
-    archiveRepository(name, repository, "")
-  })
-
-  get("/:owner/:repository/archive/*/:name")(referrersOnly { repository =>
-    val name = params("name")
-    val path = multiParams("splat").head
-    archiveRepository(name, repository, path)
+  get("/:owner/:repository/archive/*")(referrersOnly { repository =>
+    val format = params("format")
```

Comment:
How about offering the previous bahvior if `format` parameter is not available for the backward compatibility? Generating a waring long in that case would be fine.